test/test_functional_tensor.py

```diff
@@ -39,13 +40,13 @@ def _test_fn_on_batch(self, batch_tensors, fn, scripted_fn_atol=1e-8, **fn_kwarg
         for i in range(len(batch_tensors)):
             img_tensor = batch_tensors[i, ...]
             transformed_img = fn(img_tensor, **fn_kwargs)
-            self.assertTrue(transformed_img.equal(transformed_batch[i, ...]))
+            assert_equal(transformed_img, transformed_batch[i, ...])
 
         if scripted_fn_atol >= 0:
             scripted_fn = torch.jit.script(fn)
             # scriptable function test
             s_transformed_batch = scripted_fn(batch_tensors, **fn_kwargs)
-            self.assertTrue(transformed_batch.allclose(s_transformed_batch, atol=scripted_fn_atol))
+            torch.testing.assert_close(transformed_batch, s_transformed_batch, rtol=1e-5, atol=scripted_fn_atol)
```

Comment:
For ref the `rtol=1e-5` comes from the current default of https://pytorch.org/docs/stable/generated/torch.allclose.html

I'd prefer to leave `rtol` to its default in `assert_close` if possible, but `rtol` must be set if `atol` is set. Would you know the reason @pmeier ? `np.testing.assert_allclose` doesn't have this constraint it seems

Reply:
We went for this logic, since we have non-zero defaults for `rtol` and `atol`. Imagine setting `atol=0` and the tensors still match because `rtol > 0`. See https://github.com/pytorch/pytorch/blob/74c12da4517c789bea737dc947d6adc755f63176/torch/testing/_asserts.py#L391-L396.